### ISSUE TITLE
More policies needed than the ones described in the README

### ISSUE THREAD

**Issue by rszumlakowski** [pivotal-cf/terraforming-aws#63]
There are a handful of policies listed in the README file that are required by users using `terraforming-aws` but they aren't enough.  I had to add a custom policy in order to successfully get `terraforming-aws` to pave the entire infrastructure it wants.

Here's the JSON for the custom policy I had to add.  I think it should be described in the README file, at least.

```
{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Sid": "VisualEditor0",
            "Effect": "Allow",
            "Action": [
                "kms:ScheduleKeyDeletion",
                "kms:DisableKey",
                "kms:UpdateKeyDescription"
            ],
            "Resource": [
                "arn:aws:kms:*:*:*/*"
            ]
        },
        {
            "Sid": "VisualEditor1",
            "Effect": "Allow",
            "Action": [
                "kms:ListKeys",
                "kms:GenerateRandom",
                "kms:ListAliases",
                "kms:CreateKey"
            ],
            "Resource": "*"
        }
    ]
}
```

**Comment by rszumlakowski** [comment]
I ran into the above error when using `terraforming-aws` 0.15.0.

**Comment by cdutra** [comment]
@rszumlakowski I was able to `terraform apply` then `terraform destroy` by adding `kms:ScheduleKeyDeletion` and `kms:UpdateKeyDescription` as described by @heycait. I'll add these two to the documentation.
Do you have any concern regarding the other permissions?

**Comment by rszumlakowski** [comment]
All right, I can confirm that I didn't need those extra permissions in my custom policy.  Just like @heycait I was able to get away with just `kms:ScheduleKeyDeletion` and `kms:UpdateKeyDescription` in my custom policy.  I didn't need `kms:DisableKey` nor any of the keys in the `VisualEditor1` block as described in my original comment above.

For what it's worth I tried this new setup with `terraforming-aws` `v0.20.0` this morning.  When I filed this bug it was on `v0.15.0`.  I'm not sure if something has changed since then in `terraforming-aws` but it looks like it's made a difference.  Additionally, I also had the following policies in this user: AmazonRDSFullAccess, AmazonEC2FullAccess, IAMFullAccess, AmazonS3FullAccess, AWSKeyManagementServicePowerUser, AmazonRoute53FullAccess.
